EESSI-pilot-install-software.sh

```diff
@@ -269,6 +269,21 @@ else
     error "Installation of TensorFlow failed, why am I not surprised..."
 fi
 
+echo ">> Installing ReFrame 3.2..."
+$EB ReFrame-3.2.eb --robot
+if [[ $? -eq 0 ]]; then
+    echo_green "ReFrame installed, enjoy!"
+else
+    error "Installation of ReFrame failed, that's a bit strange..."
+fi
+
+echo ">> Installing RStudio-Server 1.3.1093..."
+$EB --from-pr 11764 RStudio-Server-1.3.1093-foss-2020a-Java-11-R-4.0.0.eb --robot
```

Comment:
@bedroge RStudio-Server has an OS dep:
```
osdependencies = [
    ('pam-devel', 'libpam0g-dev')
]
``` 
Is that already available in the prefix layer?